### PR TITLE
Show diffs between production and staging sites

### PR DESCRIPTION
**Pull request by ghost**
Provide a rake task that will produce visual diffs and html diffs of the most popular pages in each format on the gov.uk production site and the staging site.

Trello: https://trello.com/c/31xND00a/407-produce-a-visual-diff-between-staging-and-production-given-a-list-of-pages-to-check

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+This app provides a rake task to produce visual diffs as screenshots, and HTML diffs, of the production GOVUK website as compared with staging. Both are viewable as browser pages.  It looks a the 10 most popular pages (this is configurable) of each document format.
+
+## Screenshots
+
+![Example output](docs/screenshots/gallery.png?raw=true "Example gallery of differing pages")
+
+
+## Technical documentation
+
+It uses a fork of the BBC's wraith gem (The fork is at https://github.com/alphagov/govuk-diff-pages, the
```
Comment (rboulton) [inline README.md]: I think the URL on this line should be https://github.com/alphagov/wraith

File: lib/link_checker.rb [hunk 2]
```diff
@@ -0,0 +1,46 @@
+require 'yaml'
+require 'rest-client'
+require_relative 'root'
+require_relative 'app_config'
+require_relative 'url_checker'
+
+
```
Comment (rboulton) [inline lib/link_checker.rb]: Extra whitespace here

File: lib/page_indexer.rb [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# This class is resonsible for getting list of the most popular pages and placing them in a
```
Comment (rboulton) [inline lib/page_indexer.rb]: typo: "responsible"

File: lib/page_searcher.rb [hunk 4]
```diff
@@ -0,0 +1,45 @@
+# This class is repsonsible for taking a list of document formats and finding the top ten
```
Comment (rboulton) [inline lib/page_searcher.rb]: type: "responsible" :)

**Comment by jackscotti** [comment]
Please add a link to the ticket (if there is one).

**Pull request merged**